### ISSUE TITLE
Use training parameters from the Faster RCNN paper

### ISSUE THREAD

**Issue by agalbachicar** [ekumenlabs/fruit_detection#22]
**Description**

Replicate the most prominent paper hyperparameter values.

https://arxiv.org/pdf/1506.01497

"We use a learning rate of 0.001 for 60k mini-batches, and 0.0001 for the next 20k mini-batches on the PASCAL VOC dataset. We use a
momentum of 0.9 and a weight decay of 0.0005 [37]. Our implementation uses Caffe"

Current values:

- learning rate: 0.001
- momentum factor: 0 -> default
- weight decay: 0 -> default

**Definition of done**

Change those values to:

- learning rate: 0.001
- momentum factor: 0.9
- weight decay: 0.0005
